package.json

```diff
@@ -6,35 +6,29 @@
   "license": "BSD-3-Clause",
   "type": "module",
   "scripts": {
-    "dev": "npm run build:local && yext pages generate-test-data && pages dev",
-    "build:local": "vite build",
-    "build": "yext pages build && yext pages generate-test-data && yext pages render",
-    "build:serve": "npm run build && yext pages serve",
-    "serve": "yext pages serve",
-    "features": "pages generate features",
-    "fmt": "prettier -w .",
-    "lint": "eslint .",
-    "clean": "rm -rf .artifact-output .yext/ dist/ localData sites-rendered-output",
-    "generate-notices": "generate-license-file --input package.json --output ./THIRD-PARTY-NOTICES --overwrite"
+    "dev": "pages dev",
+    "prod": "pages prod",
+    "build": "pages build"
   },
   "engines": {
-    "node": ">=17"
+    "node": "18 || 20"
   },
   "dependencies": {
     "@yext/react-components": "^0.0.5",
     "clsx": "^1.2.1",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "tailwind-merge": "^1.6.2"
+    "tailwind-merge": "^1.6.2",
+    "@yext/sites-components": "1.0.0-rc.5"
   },
   "devDependencies": {
     "@tailwindcss/typography": "^0.5.9",
     "@types/react": "^17.0.49",
     "@types/react-dom": "^17.0.17",
     "@typescript-eslint/eslint-plugin": "^5.36.2",
     "@typescript-eslint/parser": "^5.36.2",
-    "@vitejs/plugin-react": "^2.1.0",
-    "@yext/pages": "1.0.0-beta.22",
+    "@vitejs/plugin-react-swc": "^3.5.0",
+    "@yext/pages": "1.0.0-rc.6",
```

Comment:
I assume you used the upgrade cmd in pagesjs?

I'd also hold off until the next rc cut because then we'll want to move people to vite 5. I suppose this could be a second PR instead though.

Reply:
Yup I did. I can hold off for now

Reply:
Use caret